### ISSUE TITLE
Export of large or hard to CSS parse .md files fails.

### ISSUE THREAD

**Issue by Snirpsi** [KosmosisDire/obsidian-webpage-export#11]
Export of large or hard to parse .md files fails when you export a full vault. It works fine if you open the file and export it as a single file. Maybe a longer delay would be required after opening the file until it has fully loaded the CSS parsing? With some very long files, it fails to export all chapters. It seems like it times out before finishing the howl file. 
Thanks for your great work!

![image](https://user-images.githubusercontent.com/30486919/211968941-761b0357-e753-455c-a8e9-80f3fbc4b69e.png)

**Comment by Snirpsi** [comment]
A link to a repository where it didn't work.
https://github.com/MShahmD/Obsidian-Playground

**Comment by Snirpsi** [comment]
I think the problem is that obsidian doesn't load the how page at once... I think it loads things dynamically  while scrolling down.

**Comment by KosmosisDire** [comment]
Yeah, but I tell the renderer to render out the whole page with the API first. I have tested and it works on long documents most of the time. There must be something that triggers it to not load the whole thing.

**Comment by KosmosisDire** [comment]
I am super busy right now with school, but I will definitely try to fix it!

**Comment by KosmosisDire** [comment]
I am not able to reproduce this at all with your provided files, nor with files with thousands of lines or with any of my own files. I haven't been able to reproduce this issue since I fixed it, but I fixed it before this issue was created so presumably you are experiencing this even with this fix. Make sure you are updated to the latest version of obsidian and of the plugin and try again.

Also which OS are you on, and could you upload a screenshot of your export settings.

Thanks for the report

**Comment by SeaMan75** [comment]
Hi to all!
I can not export file. RSA.zip is result of export.
RSA.md.zip - zipped md file
Image - result in browser...

[RSA.zip](https://github.com/KosmosisDire/obsidian-webpage-export/files/10582780/RSA.zip)
![image](https://user-images.githubusercontent.com/15889414/216710253-df8e091c-fe88-46fb-9c5f-8a5672e61e1b.png)

[RSA.md.zip](https://github.com/KosmosisDire/obsidian-webpage-export/files/10582796/RSA.md.zip)

**Comment by KosmosisDire** [comment]
@SeaMan75 Hello, your file works for me. It alsso doesn't seem to be the same issue described here, because it doesn't conatin any of the generative element, and it doesn't have the first part of the document. 

![image](https://user-images.githubusercontent.com/39423700/216727010-64616822-bb30-4337-a282-218d4788c1a6.png)

Because this is a different issue, could you open a new issue for this? In your issue describe the following:

Which OS are you on? Can you send a screenshot of your export settings? And finally can you open the console (Ctrl + Shift + I), clear the console, and then export your file. Then send in a screenshot of the console after the export. Thanks!

Also could you try reinstalling the plugin, and see if that helps anything?

**Comment by KosmosisDire** [comment]
I am going to go ahead and close this issue, but please reopen it @Snirpsi if you are still experiencing this. And @SeaMan75 please open a new issue if you could. Thanks!

**Comment by SeaMan75** [comment]
> @SeaMan75 Hello, your file works for me. It alsso doesn't seem to be the same issue described here, because it doesn't conatin any of the generative element, and it doesn't have the first part of the document.
> 
> ![image](https://user-images.githubusercontent.com/39423700/216727010-64616822-bb30-4337-a282-218d4788c1a6.png)
> 
> Because this is a different issue, could you open a new issue for this? In your issue describe the following:
> 
> Which OS are you on? Can you send a screenshot of your export settings? And finally can you open the console (Ctrl + Shift + I), clear the console, and then export your file. Then send in a screenshot of the console after the export. Thanks!
> 
> Also could you try reinstalling the plugin, and see if that helps anything?

Thank you very much! I will open a new issue in few time!
